pypowsybl/security.py

```diff
@@ -4,38 +4,38 @@
 # iicense, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 #
-from typing import Union, Dict
+from typing import Union as _Union, Dict as _Dict
 from pypowsybl.util import create_data_frame_from_series_array as _create_data_frame_from_series_array
 
-from typing import List
+from typing import List as _List
 
 import _pypowsybl
-from _pypowsybl import ContingencyResult
-from _pypowsybl import LimitViolation
-from _pypowsybl import ContingencyContextType
-from pypowsybl.network import Network
+from _pypowsybl import ContingencyResult as _ContingencyResult
+from _pypowsybl import LimitViolation as _LimitViolation
+from _pypowsybl import ContingencyContextType as _ContingencyContextType
+from pypowsybl.network import Network as _Network
 from pypowsybl.loadflow import Parameters
```

Comment:
That one should not be exposed here, it's already available in `pypowsybl.loadflow`.